RegressionModels/LinearRegression/HousePriceEstimation.py

```diff
@@ -0,0 +1,174 @@
+import pandas as pd
+from sklearn.linear_model import LinearRegression
+import numpy as np
+from sklearn.metrics import mean_squared_error
+
+#file path of the data
+file_path = "/home/panther/Downloads/train.csv"
```

Comment:
Try using path lib module to make the path absolute, so that program can automatically figure out the csv file location instead of you manually passing exact path